impeller/renderer/backend/metal/formats_mtl.h

```diff
@@ -159,6 +159,9 @@ constexpr MTLPrimitiveType ToMTLPrimitiveType(PrimitiveType type) {
       return MTLPrimitiveTypeLineStrip;
     case PrimitiveType::kPoint:
       return MTLPrimitiveTypePoint;
+    case PrimitiveType::kTriangleFan:
+      // UNSUPPORTED
```

Comment:
Can we update this comment to say that callers are expected to perform a cap check.

Reply:
Done